Purchases/Public/Errors/ErrorCode.swift

```diff
@@ -147,6 +147,15 @@ extension ErrorCode: DescribableError {
 
 }
 
+extension ErrorCode: CustomNSError {
```

Comment:
Style nit: new line before opening and closing braces

Reply:
Thanks I'm still not used to it.